dbaas/drivers/mongodb.py

```diff
@@ -430,14 +430,17 @@ def set_configuration(self, instance, name, value):
         client = self.get_client(instance)
 
         if name == 'quiet':
-            if value.lower() == 'true':
-                value = True
+            if value == 'true':
```

Comment:
This value can be another options ? like yes, 1, etc
u can do value = bool(1 if value.lower() in ['true', 'yes', 1] else 0)
OR 
value = bool(1 if value.lower() == 'true' else 0)
With this u can remove the false logic